packages/babel-types/src/definitions/core.js

```diff
@@ -424,10 +424,15 @@ defineType("MemberExpression", {
       validate: assertNodeType("Expression"),
     },
     property: {
-      validate(node, key, val) {
-        const expectedType = node.computed ? "Expression" : "Identifier";
-        assertNodeType(expectedType)(node, key, val);
-      },
+      validate: (function () {
+        const normal = assertNodeType("Identifier");
+        const computed = assertNodeType("Expression");
```

Comment:
Probably unnecessary but we could just hoist and reuse `assertNodeType("Expression")` everywhere right? Or just precompute everything (I guess that's what prepack would do)